.github/workflows/python-dockers.yml

```diff
@@ -13,29 +13,40 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-24.04
-          - ubuntu-24.04-arm
-          # TODO: currently impossible(?) to build Docker images on macOS runners
-          #
-          # "Nested virtualization" is only supported on macOS-15 and ≥M3, but the current macos-15 runners
-          # [only have M1][^1] (or [possibly M2 Pro?][^2]) chips.
-          #
-          # [^1]: https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
-          # [^2]: https://github.com/douglascamata/setup-docker-macos-action/issues/35#issuecomment-2384104416:
-          #
+          - runs-on: ubuntu-24.04
+            suffix: ''
+          - runs-on: ubuntu-24.04-arm
+            suffix: ' (ARM)'
+          # TODO: currently impossible(?) to build Docker images on macOS runners: https://github.com/single-cell-data/TileDB-SOMA/issues/3781
           # - macos-15
-        name:
-          - pypi
-          - src
-          - langchain
-          - bookworm
-    name: ${{ matrix.os }}-${{ matrix.name }}
-    runs-on: ${{ matrix.os }}
+        img:
+          - name: ubuntu-src
+          - name: ubuntu
+            args: v=1.15.7
```

Comment:
We don't use `v` in any TileDB-SOMA versions ([procedure](https://github.com/single-cell-data/TileDB-SOMA/wiki/Branches-and-releases), [releases](https://github.com/single-cell-data/TileDB-SOMA/releases)). I think it's confusing to include them here ...